docs/blog/posts/2024/srlinux-asymmetric-routing.md

```diff
@@ -0,0 +1,1801 @@
+                admin-state enable
+                address 172.16.20.254/24 {
+                    anycast-gw true
+                }
+                arp {
+                    learn-unsolicited true
+                    host-route {
+                        populate dynamic {
+                        }
+                        populate evpn {
+                        }
+                    }
+                    evpn {
+                        advertise dynamic {
+                        }
+                    }
+                }
+            }
+            anycast-gw {
+                anycast-gw-mac 00:00:5E:00:53:00
+            }
+        }
+    }
+```
+
+///
+
+There is a lot going on here, so let's breakdown some of the configuration options:
+
+`anycast-gw [true|false]`
+
+:   When this is set to `true`, the IPv4 address is associated to the anycast gateway MAC address and this MAC address is used to respond to any ARP requests for that IPv4 address. This also allows the same IPv4 address to be configured on other nodes for the same broadcast domain, essentially suppressing duplicate IP detection.
+
+`anycast-gw anycast-gw-mac [mac-address]`
+
+:   The MAC address configured with this option is the anycast gateway MAC address and is associated to the IP address for that subinterface. If this is ommitted, the anycast gateway MAC address is auto-derived from the VRRP MAC address group range.
+
+`arp learn-unsolicited [true|false]`
+
+:   This enables the node to learn the IP-to-MAC binding from any ARP packet and not just ARP requests.
+
+`arp host-route populate [dynamic|static|evpn]`
+
+:   This enables the node to insert a host route (/32 for IPv4 and /128 for IPv6) in the routing table from dynaimc, static or EVPN-learnt ARP entries.
+
+`arp evpn advertise [dynamic|static]`
+
+:   This enables the node to advertise EVPN Type-2 MAC+IP routes from dynamic or static ARP entries.
+
+### MAC VRFs on leafs
+
+Finally, MAC VRFs are created on the leafs to create a broadcast domain and corresponding bridge table for Layer 2 learning. Since, by default, a MAC VRF corresponds to a single broadcast domain and bridge table, we can map only one Layer 2 VNI to it. Thus, on leaf1 and leaf4, two MAC VRFs are created - one for VNI 10010 and another for VNI 10020. Under the MAC VRF, there are several important things to consider:
+
+* The Layer 2 subinterface is bound to the MAC VRF using the `interface` configuration option.
+* The corresponding IRB subinterface is bound to the MAC VRF using the `interface` configuration option.
+* The VXLAN tunnel subinterface is bound to the MAC VRF using the `vxlan-interface` configuration option.
+* BGP EVPN learning is enabled for the MAC VRF using the `protocols bgp-evpn` hierarchy and the MAC VRF is bound to an EVI (EVPN virtual instance).
+* The `ecmp` configuration option determines how many VTEPs can be considered for load-balancing by the local VTEP (more on this in the validation section).
```

Comment:
form Jorge

please mention this ecmp refers to the overlay ecmp-set for multihoming aliasing (so that people do not confuse with the underlay ecmp setting)

Reply:
modified.